.idea/misc.xml

```diff
@@ -25,7 +25,11 @@
       </value>
     </option>
   </component>
+<<<<<<< HEAD
```

Comment:
I think this got overlooked while merging

Reply:
fixed